doc/installing-spacegraphcats.md

```diff
@@ -36,13 +36,19 @@ conda activate sgc
 
 and voila, done!
 
-## 2. Installing dependencies in a virtual environment
+### 2. Installing dependencies in a virtual environment
 
 If you already have a functioning Python >= 3.5 along with a C/C++
 development environment, you can install the dependencies in a
 venv.
 
-Change to a working directory, and create a virtualenv:
+Install virtualenv if you have not done so:
+
+```
+pip install virtualenv
+```
+
+Change to a working directory, and create a virtualenv (replace `python3.5` with your Python version):
```

Comment:
right! we should probably move to python3.7 or above, actually, since I think I started using f-strings. Oops.

Reply:
punted to #295